### PR TITLE
Fix images not loading in books when using a base url

### PR DESCRIPTION
**Pull request by Fesaa**
# Fixed
- Fixed: Fixes images not loading in books when using a custom base url

The way baseUrl was constructed did not take the configuration option into account, this caused all requests to fail. I didn't notice any style missing, but looking at the code; this would also fix that.

<details>
<summary> Before</summary>
image
</details>
<details>
<summary> After</summary>
image
</details>

### CODE REVIEW

File: API/Controllers/BookController.cs [hunk 1]
```diff
@@ -153,7 +153,9 @@ public async Task<ActionResult<string>> GetBookPage(int chapterId, [FromQuery] i
         if (chapter == null) return BadRequest(await _localizationService.Translate(User.GetUserId(), "chapter-doesnt-exist"));
         var path = _cacheService.GetCachedFile(chapter);
 
-        var baseUrl = "//" + Request.Host + Request.PathBase + "/api/";
+        string basePath = Configuration.BaseUrl;
```
Comment (majora2007) [inline API/Controllers/BookController.cs]: Please stick to coding style of the codebase. We use var and string.Empty. 

This line here is not needed. You would rewrite this as:
`var basepath = Configuration.BaseUrl == Configuration.DefaultBaseUrl ? string.Empty : Configuration.BaseUrl;`

**Review by majora2007 — CHANGES_REQUESTED** [review]
I will test this out. Last time I had checked, PathBase has the base url string in it.

File: API/Controllers/BookController.cs (hunk 1 above)
Comment (Fesaa) [reply API/Controllers/BookController.cs]: Done :)

**Comment by Fesaa** [comment]
Hi, sorry for the trouble. Hope you haven't tried testing this yet, I did some dumb stuff in my nginx config which broke it. Fixing this, fixed it as well :pensive:

**Comment by majora2007** [comment]
Haha no problem. Glad you got it sorted.

**Pull request closed without merging**